### ISSUE TITLE
Washing machine remote start of a specific program

### ISSUE THREAD

**Issue by vring0** [ollo69/ha-smartthinq-sensors#292]
**Is your feature request related to a problem? Please describe.**
Yes, I need help with the remote start of the desired program, if this function exists.

**Describe the solution you'd like**
send program name to service `smartthinq_sensors.remote_start`

**Additional context**

Model F_R7_Y___W.A__QEUK-FL (WASHER)
LG

**Comment by Xlinx64** [comment]
This would be very useful

**Comment by purcell-lab** [comment]
I have a Service call that does exactly this, I was originally looking for a switch, but the Service Call can be called from automations.

I now load the washing machine and hit the remote start button, the washing machine sits idle/ sleeps, and I call the washer to remote start when I am generating excess solar.

![Screenshot 2022-04-05 10 11 03](https://user-images.githubusercontent.com/79175134/161653810-78825b89-24af-4878-9120-e5da1d4ae169.png)

**Comment by vring0** [comment]
> I have a Service call that does exactly this, I was originally looking for a switch, but the Service Call can be called from automations.
> 
> I now load the washing machine and hit the remote start button, the washing machine sits idle/ sleeps, and I call the washer to remote start when I am generating excess solar.
> 
> ![Screenshot 2022-04-05 10 11 03](https://user-images.githubusercontent.com/79175134/161653810-78825b89-24af-4878-9120-e5da1d4ae169.png)

Hello! 
Thanks, but I need to configure the mode before running the program. For example: Cotton 90°C, spin 1000, etc.

**Comment by Bridelance** [comment]
Works like a charm!! :)

However, in the LGThinq app there is the option to pause the cycle, any chance this can be added as well?

This would be an awesome feature if you want to control your dryer/washing machine by solar power: turn on dryer when having a lot of sun shine, pause when there is a sudden rain shower ...

**Comment by michelebossa** [comment]
Hello,
yes i'm interesting to Pause features to avoid power disconnection due over energy consuming will be very helpful.

Thanks.

**Comment by Xlinx64** [comment]
I would still be happy to select a programm via home assistant

**Comment by vring0** [comment]
> Я бы еще с удовольствием выбрал программу через домашнего помощника

+1

**Comment by idorose** [comment]
what program does it run when you use this service? is it what was last highlighted with the selector?
in the app you need to wake it up and choose a program.

**Comment by ollo69** [comment]
> what program does it run when you use this service? is it what was last highlighted with the selector?
> in the app you need to wake it up and choose a program.

I do not really use this feature, but when I released this was the way it works. Add the possibility to provide the program to use is in my queue, but priority is not very high.

**Comment by lcavalli** [comment]
I would keep this feature request open, since it is in the @ollo69 's todo queue.

**Comment by Shogun1978** [comment]
Would be really great to see this feature. I tested around a little bit. So every time I set the programm I want for washing and the maschine wents to "sleep", it forgets the choosen program after waking up. So I need the app to set the correct program.

It would be much easier to set this program using home assistant as long as LG doesn't remind the program that was set before the sleep mode. I'm aiming to start the washing maschine when the solar energy is at peak hour.

**Comment by mijeca** [comment]
> I want for washing and the maschine wents to "sleep", it forgets the choosen program after waking up.

Yep. THat happen with my washer and dryer,  both from LG, and as far as i could unserstand, both using lg api v1.

**Comment by slickers74** [comment]
I did try a wake comand and then remote start but it defaults to the cotton wash cycle.
I was trying to remote start the dry function of the machine at midnight when I'm on a cheaper electricity tarrif.

**Comment by guillaumeharvey26** [comment]
Anyone have a solution to select a program before remote start?  In my case, I want to select "Super Dry" program just before to start!

**Comment by ollo69** [comment]
This is a very old request, patience will be rewarded😉
In last release now is possible to select a program (standard program, not smartcouse / downloadcourse) to use for remote start. 
Please try and let me know.

**Comment by Xlinx64** [comment]
Thank you so much, I tried it today and it worked like a charm. I sent some coffe your way, cheers!
The only thing that I would maybe wish for the future is a selector for RPM and temperature.

**Comment by ollo69** [comment]
Ok, I will think about this, but send additional information should be quite complex because this depend on the program selected and the device. It could be additional info to provide to the service call but not managed with the UI, because validate all the combination could be a mess.

**Comment by slickers74** [comment]
Thats a nice notification to wake up to - Thanks for looking at this, I'll give it a try after work.

**Comment by slickers74** [comment]
Works great! Thanks for this.
